TestCases/serial_regression.py

```diff
@@ -1030,6 +1030,18 @@ def main():
 #    pass_list.append(findiff_euler_py.run_filediff())
 #    test_list.append(findiff_euler_py)
     
+#    # test finite_difference.py
```

Comment:
Please remove these lines if we will not be using the finite differences case in the end.